canonical-web-and-design/snapcraft.io
# Update eslint and node-sass to latest versions

## QA

- Pull the branch
- Ensure your dev environment still works ok
- Ensure building, linting and testing still work ok
